.github/ISSUE_TEMPLATE/breaking-change.yml

```
@@ -0,0 +1,107 @@
+name: ".NET breaking change"
+desription: Report a change in .NET that breaks something that worked in a previous version. Intended mostly for product-team use.
+title: "[Breaking change]: "
+labels: breaking-change
+assignees:
+ - gewarren
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Brief description of the breaking change.
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of .NET introduced the breaking change?
+      options:
+        - .NET 6 Preview 6
+        - .NET 6 Preview 7 (Default)
+        - .NET 6 Preview 8
+        - .NET 6 Preview 9
+        - .NET 6 Preview 10
+        - .NET 6 GA
+        - Other (please put exact version in description textbox)
+    validations:
+      required: true
+  - type: textarea
+    id: old-behavior
+    attributes:
+      label: Previous behavior
+      description: Describe the previous behavior.
+    validations:
+      required: true
+  - type: textarea
+    id: new-behavior
+    attributes:
+      label: New behavior
+      description: Describe the new behavior. Include code snippets if applicable.
+    validations:
+      required: true
+  - type: checkboxes
+    id: change-type
+    attributes:
+      label: Type of breaking change
+      description: Does this change affect binary compatibility, source compatibility, or both?
+      options:
+        - label: Binary compatibility (code must be recompiled to use the newer API version)
+        - label: Source compatibility (successfully recompiling against the newer API version requires source changes)
+    validations:
+      required: true
+  - type: textarea
+    id: reason
+    attributes:
+      label: Reason for change
+      description: Describe why the breaking change was introduced.
+    validations:
+      required: true
+  - type: textarea
+    id: recommended-action
+    attributes:
+      label: Recommended action
+      description: Describe the recommended action an affected user should take, such as workarounds or examples of code changes.
+    validations:
+      required: true
+  - type: dropdown
+    id: area
+    attributes:
+      label: Feature area
+      description: What feature area(s) does the change affect?
+      options:
+        - .NET 6 Preview 6- [ ] ASP.NET Core
```

This looks like a typo?

```suggestion
        - .NET 6 Preview 6
        - ASP.NET Core
```